src/components/type.js

```diff
@@ -2,7 +2,7 @@ import { useEffect, useMemo, useRef, useState } from 'react';
 import styled from 'styled-components';
 
 const interval = 100;
-const idle = 1000;
+const idle = 1;
 
 const srOnly = {
```

Comment:
Can we use a styled component for this?

Reply:
Can we make the screen-reader text a styled component too, while we're here, please?

Reply:
done, and sorry I forgot to check cross-browser compatibility
I also fixed the issue the best I could
I have tested it on chromium and firefox